### PR TITLE
Added some objective C into the project and tried to add a list feature.

### PR DESCRIPTION
**Pull request by XanderSchoeman**
Some basic objective c stuff was created and testing so that it creates a message in objective c which is used to display in a alert in swift when you press the favourite button open choosing an anime. Also started with the Favourites screen.

Favourite button:
Screenshot 2020-03-24 at 08 40 58
Button pressed:
Screenshot 2020-03-24 at 08 41 07

### CODE REVIEW

File: XouDevSpec/Classes/Helper/ObjCAnimeClass.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+@property NSString *episodes;
+@property NSString *score;
+@property NSString *start_date;
+@property NSString *end_date;
+@property NSString *members;
+@property NSString *rated;
+
+-(NSString*)AnimeInfoString;
+
+
```
Comment (AMadanlal) [inline XouDevSpec/Classes/Helper/ObjCAnimeClass.h]: Need to remove open spaces in code

Comment (XanderSchoeman) [reply XouDevSpec/Classes/Helper/ObjCAnimeClass.h]: kk

File: XouDevSpec/Classes/Helper/ObjCAnimeClass.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//
+//  Created by Xander Schoeman on 2020/03/22.
+//  Copyright © 2020 Xander Schoeman. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ObjCAnimeClass : NSObject
+-(instancetype)initWith:(NSString *)mal_id :(NSString *)url :(NSString *)image_url :(NSString *)title :(NSString *)airing :(NSString *)synopsis :(NSString *)type :(NSString *)episodes :(NSString *)score :(NSString *)start_date :(NSString *)end_date :(NSString *)members :(NSString *)rated;
```
Comment (AMadanlal) [inline XouDevSpec/Classes/Helper/ObjCAnimeClass.h]: This could be a struct

**Comment by XanderSchoeman** [comment]
Cool cool thank you and your comments are taken into account, I just have to figure out hwy bitrise is breaking with the main project now xD

File: XouDevSpec/Classes/Helper/ObjCAnimeClass.h (hunk 2 above)
Comment (Zamo22) [reply XouDevSpec/Classes/Helper/ObjCAnimeClass.h]: Definitely, that's a massive init lol

Comment (XanderSchoeman) [reply XouDevSpec/Classes/Helper/ObjCAnimeClass.h]: hehe, I actually tried to use it differently but it seems that I might just remove it and remake the whole objective c thing xD. But yeah you both make a good point it was also a pain typing it haha.

**Pull request merged**